tools/gradle/commons/integrations/python.gradle

```diff
@@ -27,11 +29,22 @@ task isortFormat(type: PythonTask) {
     command = ". --settings-file ${rootProject.file('tools/python/.isort.cfg').absolutePath}"
 }
 
+task docFormat(type: PythonTask) {
+    module = "docformatter"
+    command = "-ri ."
+}
+
 task flakeCheck(type: PythonTask, dependsOn: blackFormat) {
     module = "flake8"
     command = ". --config ${rootProject.file('tools/python/.flake8').absolutePath}"
 }
 
+task pylintCheck(type: PythonTask, dependsOn: blackFormat) {
+    module = "pylint"
+    // FIXME this is not working yet...
+    command = "--rcfile=${rootProject.file('tools/python/pylintrc').absolutePath}"
+}
+
 if(project.ext.has('pyModule')) {
```

Comment:
I think this is the sort of guard necessary for pylint? The folder/module name passed in with `pyModule` (example: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/bases/base-singer/build.gradle#L1)